website/blog/2023-04-14-release-0.14.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Release Notes - Podman Desktop 0.14
+description: Podman Desktop 0.14 has been released! 
+slug: podman-desktop-release-0.14
+authors: [deboer]
+tags: [podman-desktop, release, kubernetes, kind]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 0.14 - Our Kind-est release yet!
+
+We have been working on a Kind extension for a while now, and decided it is time to promote it
+into the release just in time for KubeCon and CloudNativeCon Europe!
+
+Some of these features were developed in the Kind extension over the last few releases,
+but since they are new to you, and they were not in previous release notes, we will do a
+roundup and talk about all the Kind features.
+
+<!--Main Features-->
+
+- **Kind Installation**: Install Kind from the status bar
+- **Kind Clusters in Resources**: Create and manage Kind clusters from Resources
+- **Sharing Images with Kind**: Easily move images to Kind
+- **Kind Ingress**: Install an ingress controller
+- **Play YAML on Kubernetes**: Deploy a YAML file to a cluster
+- **UX and UI Improvements**: Updated preferences and telemetry prompt
+
+Podman Desktop 0.14 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-0-14-hero](img/podman-desktop-release-0.14/podman-desktop-release-0.14.png)
+
+<!--truncate-->
+_________________
+
+## Release details
+
+### Kind Installation
+
+The Kind extension is now bundled as part of Podman Desktop [1421](https://github.com/containers/podman-desktop/issues/1421)
+and allows you to easily [install Kind directly from the status bar](/docs/kubernetes/kind/installing-kind) [1257](https://github.com/containers/podman-desktop/issues/1257).
+The installed `kind` CLI is available from the system shell [1516](https://github.com/containers/podman-desktop/issues/1516).
+
+### Kind Clusters in Resources
+
+[Create Kind clusters from an updated **Settings > Resources** page](/docs/kubernetes/kind/creating-a-kind-cluster) that also allows you to start/stop [1953](https://github.com/containers/podman-desktop/issues/1953) and delete [1977](https://github.com/containers/podman-desktop/issues/1977) them.
+
+### Share Images with Kind
+
+It is easy to move images from Podman to Kind [1448](https://github.com/containers/podman-desktop/issues/1448).
+
+### Ingress
+
+If you deploy a pod to Kind, you are going to want to reach it! We have made it easy to install an ingress
```

Comment:
reaching out to the cluster or reaching out to the application running in the cluster?
How did we enable that? How the user/reader can use it?